### PR TITLE
Better highlighting of archved labels

### PR DESCRIPTION
**Pull request by 6543**
as followup of the not jet finished discussion at https://github.com/go-gitea/gitea/pull/29680#discussion_r1521867261

we enhance and chat about how best to highlight archived labels here :)

### CODE REVIEW

**Comment by delvh** [comment]
Could you post a screenshot?

**Comment by silverwind** [comment]
Maybe opacity alone is enough, not sure. Some screenshots of the variants would help.

**Comment by silverwind** [comment]
opacity .5:

Screenshot 2024-03-15 at 01 01 52

opacity .5, greyscale .5:

Screenshot 2024-03-15 at 01 01 31

opacity .5, greyscale .8:

Screenshot 2024-03-15 at 01 03 21


I think I prefer the second, preserves a bit of the original label coloring.

File: web_src/css/repo.css [hunk 1]
```diff
@@ -2419,6 +2419,7 @@
 
 .archived-label {
   filter: grayscale(0.8);
```
Comment (silverwind) [inline web_src/css/repo.css]:
```suggestion
  filter: grayscale(0.5);
```

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by 6543** [comment]
better idea:  #30376

**Pull request merged**